suite/contracts/contracts_runlog_test.go

```diff
@@ -18,87 +18,102 @@ import (
-		var jobID [32]byte
-		copy(jobID[:], jobUUIDReplaces)
-		err = consumer.CreateRequestTo(
-			s.Wallets.Default(),
-			oracle.Address(),
-			jobID,
-			big.NewInt(1e18),
-			adapter.ClusterURL()+"/five",
-			"data,result",
-			big.NewInt(100),
-		)
-		Expect(err).ShouldNot(HaveOccurred())
-		err = retry.Do(func() error {
-			d, err := consumer.Data(context.Background())
-			if d == nil {
-				return errors.New("no data")
-			}
-			log.Debug().Int64("Data", d.Int64()).Send()
-			if d.Int64() != 5 {
-				return errors.New("data is not on chain still")
-			}
-			if err != nil {
-				return err
+		By("Funding Chainlink nodes", func() {
+			nodes, err = environment.GetChainlinkClients(s.Env)
+			Expect(err).ShouldNot(HaveOccurred())
+			nodeAddresses, err = actions.ChainlinkNodeAddresses(nodes)
+			Expect(err).ShouldNot(HaveOccurred())
+			err = actions.FundChainlinkNodes(nodes, s.Client, s.Wallets.Default(), big.NewInt(2e18), nil)
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+		By("Deploying and funding the contracts", func() {
+			oracle, err = s.Deployer.DeployOracle(s.Wallets.Default(), s.Link.Address())
+			Expect(err).ShouldNot(HaveOccurred())
+			consumer, err = s.Deployer.DeployAPIConsumer(s.Wallets.Default(), s.Link.Address())
+			Expect(err).ShouldNot(HaveOccurred())
+			err = consumer.Fund(s.Wallets.Default(), nil, big.NewInt(2e18))
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+		By("Permitting node to fulfill request", func() {
+			err = oracle.SetFulfillmentPermission(s.Wallets.Default(), nodeAddresses[0].Hex(), true)
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+		By("Creating directrequest job", func() {
+			jobUUID = uuid.NewV4()
+			os := &client.DirectRequestTxPipelineSpec{
+				URL:         adapter.ClusterURL() + "/five",
+				Method:      "GET",
+				RequestData: "{}",
+				DataPath:    "data,result",
 			}
-			return nil
+			ost, err := os.String()
+			Expect(err).ShouldNot(HaveOccurred())
+			_, err = nodes[0].CreateJob(&client.DirectRequestJobSpec{
+				Name:              "direct_request",
+				ContractAddress:   oracle.Address(),
+				ExternalJobID:     jobUUID.String(),
+				ObservationSource: ost,
+			})
+			Expect(err).ShouldNot(HaveOccurred())
 		})
-		Expect(err).ShouldNot(HaveOccurred())
-	},
-		Entry(
-			"on Ethereum Hardhat",
-			environment.NewChainlinkCluster("../../", 1),
-			client.NewNetworkFromConfig,
-			contracts.DefaultFluxAggregatorOptions(),
-		),
-	)
+		By("Calling oracle contract", func() {
+			jobUUIDReplaces := strings.Replace(jobUUID.String(), "-", "", 4)
+			Expect(err).ShouldNot(HaveOccurred())
+			var jobID [32]byte
+			copy(jobID[:], jobUUIDReplaces)
+			err = consumer.CreateRequestTo(
+				s.Wallets.Default(),
+				oracle.Address(),
+				jobID,
+				big.NewInt(1e18),
+				adapter.ClusterURL()+"/five",
+				"data,result",
+				big.NewInt(100),
+			)
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+	})
+
+	Describe("Check directrequest job", func() {
```

Comment:
Small comment, but the description of steps with describe should not be directly stating "Check" or "Verify" etc. It should be like:
```
with DirectRequest job
```
then:
```
receives API call data on-chain
```

One weird thing I noticed in the Ginkgo docs is that the `By` calls all capitalised the first letter, whereas `Describe` and `It` didn't.